widget/info.json

```diff
@@ -1,15 +1,13 @@
 {
     "name": "heatmap",
     "title": "Heatmap",
-    "subTitle": "Show Fortisoar data having 2D property using heatmap",
+    "subTitle": "Heatmap widget creates division of data using different color codes",
```

Comment:
suggested text: A heatmap widget is used to visually represent data where values are depicted by color, allowing users to quickly identify patterns and trends across multiple variables by seeing which areas on a grid have the highest concentration of a specific value.